modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-major.adoc

```diff
@@ -0,0 +1,359 @@
+        circuitBreaker="GlobalCircuit"/>
+    <flow-ref name="invoiceProcess">
+</flow>
+
+<flow name="otherSubscribe">
+    <anypoint-mq:subscriber
+        destination="${paymentsQueue}"
+        config-ref="Anypoint_MQ_Config"
+        circuitBreaker="GlobalCircuit"/>
+    <flow-ref name="invoiceProcess">
+</flow>
+
+<sub-flow name="invoiceProcess">
+  <ftp:write path="${auditFolder}" config-ref="ftp-config"/>
+  <http:request config-ref="requestConfig" path="/external"/>
+</sub-flow>
+
+----
+
+=== Publish Operation
+
+In version 2.x of this connector, `Publish` operation properties were declared like this:
+
+[source,xml,linenums]
+----
+<anypoint-mq:publish config-ref="Anypoint_MQ_Default_subscriber"
+       destination="queue"
+       messageId="#[vars.messageId]" sendContentType="false">
+   <anypoint-mq:body >#[vars.messageBody]</anypoint-mq:body>
+   <anypoint-mq:properties >
+      <anypoint-mq:property key="MSG_TYPE" value="My Value"/>
+   </anypoint-mq:properties>
+</anypoint-mq:publish>
+----
+
+In version 3.x, you now declare the same properties as a dynamic map, subject to transformations and a dynamic number of keys, instead of having fixed keys for each message:
+
+[source,xml,linenums]
+----
+<anypoint-mq:publish config-ref="Anypoint_MQ_Config"
+     destination="queue"
+     messageId="#[vars.currentId]" sendContentType="false">
+   <anypoint-mq:body >#[vars.messageBody]</anypoint-mq:body>
+   <anypoint-mq:properties ><![CDATA[#[output application/java -----
+{
+   "MSG_TYPE" : vars.msgType
+}]]]></anypoint-mq:properties>
+</anypoint-mq:publish>
+----
+
+The following table describes the `Publish` operation differences.
+
+[%header,cols="35,30,30"]
+|===
+| Changes | Version 2.x | Version 3.x
+| Execution Type
+| Blocking
+| Non-Blocking
+
+| Properties Parameter
+| Fixed key/value map
+| Dynamic "content" parameter map
+|===
+
+
+For more information, see xref:anypoint-mq/3.x/anypoint-mq-publish.adoc[Anypoint MQ Publish Operation]
+
+=== Consume Operation
+
+The following table describes the Consume operation differences.
+
+[%header,cols="35,30,30"]
+|===
+| Changes | Version 2.x | Version 3.x
+| Execution Type
+| Blocking
+| Non-Blocking
+
+| Default Acknowledgment Mode
+| MANUAL
+| IMMEDIATE
+|===
+
+For more information, see xref:anypoint-mq/3.x/anypoint-mq-consume.adoc[Anypoint MQ Consume Operation].
+
+=== ACK and NACK Operations
+
+The ACK and NACK operations now use the `ackToken` string.
+
+You can obtain this new `ackToken` value from the attributes object. The `ackToken` is available only for messages that have a `MANUAL` acknowledgment mode.
+
+The following table describes the ACK and NACK operation differences.
+
+[%header,cols="50,50"]
+|===
+| Version 2.x | Version 3.x
+| Operations received a `messageContext` parameter, whose value had to be the `attributes` element from a given message.
+| Use the `ackToken` string of the message to perform the ACK or NACK of a message.
```

Comment:
Just to beat this dead horse a bit more.... Avoid using imperative voice outside of steps.